src/scripts/calibration_analyses/analysis_scripts/analysis_demography_calibrations.py

```diff
@@ -102,9 +102,9 @@ def apply(results_folder: Path, output_folder: Path, resourcefilepath: Path = No
             marker='o', markersize=10, linestyle='none', label='Census', zorder=10, color=colors['Census'])
     ax.plot(pop_model.index, pop_model['mean'] / 1e6,
             label='Model (mean)', color=colors['Model'])
-    ax.fill_between((pop_model.index).to_numpy(),
-                    (pop_model['lower'] / 1e6).to_numpy(),
-                    (pop_model['upper'] / 1e6).to_numpy(),
+    ax.fill_between((pop_model.index).to_numpy(dtype=float),
+                    (pop_model['lower'] / 1e6).to_numpy(dtype=float),
+                    (pop_model['upper'] / 1e6).to_numpy(dtype=float),
```

Comment:
This can probably also be fixed by changing how the `pop_model` dataframe is constructed (at the moment the columns are `object`s, and they can't be properly be used inside `ax.fill_between`), but coercing conversion to `float` looked like a quick solution without too much refactoring.

Reply:
This actually started working to me without changes in the code, once I merged the Fix to summarize [#1150].

Reply:
This also worked for me.